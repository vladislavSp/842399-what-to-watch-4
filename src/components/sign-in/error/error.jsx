import React from 'react';

export const ErrorLogin = () => (<div className="sign-in__message">
  <p>Please enter a valid email address</p>
</div>);

export const ErrorPass = () => (<div className="sign-in__message">
  <p>We canâ€™t recognize this email <br /> and password combination. Please try again.</p>
</div>);
