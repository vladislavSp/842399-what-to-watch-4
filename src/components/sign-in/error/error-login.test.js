import React from "react";
import renderer from "react-test-renderer";
import {ErrorLogin} from "./error.jsx";

it(`ErrorLogin render correctly`, () => {
  const tree = renderer
    .create(
        <ErrorLogin />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
