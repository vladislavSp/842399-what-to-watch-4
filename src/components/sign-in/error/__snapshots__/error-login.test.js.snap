// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLogin render correctly 1`] = `
<div
  className="sign-in__message"
>
  <p>
    Please enter a valid email address
  </p>
</div>
`;
